packages/suite/src/views/recovery/index.tsx

```diff
@@ -305,7 +306,17 @@ const Recovery = ({
                         </StyledP>
                         <StyledImage image="UNI_SUCCESS" />
                         <Buttons>
-                            <CloseButton onClick={() => closeModalApp()} />
+                            <CloseButton
+                                onClick={() => {
+                                    closeModalApp();
```

Comment:
wouldnt be better to create one function for that, like `onSetRecoveryType` (line 128) with a success: boolean parameter and use it in both cases, here and below?